pkg/api/norman/customization/cluster/validator.go

```diff
@@ -164,7 +164,9 @@ func (v *Validator) validateLocalClusterAuthEndpoint(request *types.APIContext,
 		}
 		isValidCluster = cluster.Status.Driver == "" ||
 			cluster.Status.Driver == v32.ClusterDriverRKE ||
-			cluster.Status.Driver == v32.ClusterDriverImported
+			cluster.Status.Driver == v32.ClusterDriverImported ||
+			cluster.Status.Driver == v32.ClusterDriverK3s ||
+			cluster.Status.Driver == v32.ClusterDriverRke2
 	}
 	if !isValidCluster {
 		return httperror.NewFieldAPIError(httperror.InvalidState, "LocalClusterAuthEndpoint.Enabled", "Can only enable LocalClusterAuthEndpoint with RKE")
```

Comment:
Should this error message be updated?

```suggestion
		return httperror.NewFieldAPIError(httperror.InvalidState, "LocalClusterAuthEndpoint.Enabled", "Can only enable LocalClusterAuthEndpoint with RKE, RKE2, or K3s")
```

or maybe "Cannot enable LocalClusterAuthEndpoint for cluster driver %s"

Reply:
Updated.